src/simpleitk_timeseries_motion_correction/create_animation.py

```
@@ -274,4 +276,8 @@ def main(input_file, output_file, additional_input_files=None, scale=2.0, fps=10
 
     args = parser.parse_args()
 
-    main(args.input, args.output, args.additional_row, args.scale, args.fps)
+    print(f"Loading input images...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)